config/plugins/interactive_environments/ipython/templates/ipython.mako

```diff
@@ -19,23 +18,15 @@ else:
     PASSWORD = "none"
 
 ## IPython Specific
-# Prepare an empty notebook
-notebook_id = ie_request.generate_hex(64)
-with open( os.path.join( ie_request.attr.our_template_dir, 'notebook.ipynb' ), 'r') as nb_handle:
-    empty_nb = nb_handle.read()
-empty_nb = empty_nb % notebook_id
-# Copy over default notebook, unless the dataset this viz is running on is a notebook
-empty_nb_path = os.path.join(temp_dir, 'ipython_galaxy_notebook.ipynb')
-
-if hda.datatype.__class__.__name__ != "Ipynb":
-    with open( empty_nb_path, 'w+' ) as handle:
-        handle.write( empty_nb )
+if hda.datatype.__class__.__name__ == "Ipynb":
+    DATASET_HID = hda.hid
```

Comment:
I complained about this elsewhere, it just ... seems odd to me. I cannot imagine that notebook sizes will ever be a problem for `get()`ing them (https://github.com/bgruening/docker-ipython-notebook/pull/63/files#r44854985) but it just feels wrong to have it not be a volume mounted `:ro`